convergence_tests/Couette.cc

```diff
@@ -4,19 +4,20 @@
  * @brief The source file solving the Couette flow problem.
  *
  */
```

Comment:
We do not really use this program anymore. We can just delete it.

Reply:
I would prefer to keep this program as a convergence test.